content/products/workspaces.md

```diff
@@ -5,20 +5,22 @@ linktitle: /products/workspaces
 date: 2021-01-08
 g3Teaser:
   title: Workspaces
-  detail: Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.
+  detail: Gen3 workspaces are secure data analysis environments in the cloud that can access data from one or more data resources, including Gen3 Data Commons. Workspaces are often fully integrated with a specific data commons, and coming soon are workspaces as stand-alone analysis environments with a user pay model.
+  para1: Gen3 Workspaces use the Gen3 Framework Services for user authentication and authorization and for retrieving data objects and metadata from data resources, like Gen3 Data Commons.
+  para2: By default Gen3 Workspaces include Jupyter notebooks and RStudio. Workspaces can be further configured to offer a variety of analysis environment "flavors", pre-loaded with various custom applications. Virtually any application can be integrated into a Gen3 Workspace, like analysis workflows, data processing pipelines, or data visualization apps.
```

Comment:
To shorten it a bit if wanted: The third section is repeated below, too:
"By default Gen3 Workspaces include Jupyter notebooks and RStudio, but virtually any application such as analysis workflows, data processing pipelines, or data visualization apps can be integrated into a Gen3 Workspace."

Reply:
done, thanks for the R